packages/invariant/index.ts

```diff
@@ -0,0 +1,29 @@
+const isProduction: boolean = process.env.NODE_ENV === "production";
+const prefix = "Visit https://community.frontity.org for help! 🙂\n\n";
+
+// Throw an error if the condition fails
+export function invariant(condition: boolean, message?: string) {
+  if (condition) {
+    return;
+  }
+  // Condition not passed
+
+  if (isProduction) {
+    throw new Error(prefix);
```

Comment:
Are we sure we want all our error messages in production to just say `"Visit https://community.frontity.org for help"`? Those are the messages that are going to be thrown in production sites, and production site users should not be redirected to our community.

Apart from that, I've just realized that there's not much point in removing the message to save Kbs because the message is not inside a `process.env.NODE_ENV === "production"` conditional and it won't be remove by Webpack.

@SantosGuillamot what do you think? What type of error messages should we show in development and production modes?

Reply:
Not sure how developer tools usually work, but I feel that messages redirecting them to the community should be only included in development mode. I will take a deeper look at other tools, but this is what React codebase says about [warnings and invariants](https://reactjs.org/docs/codebase-overview.html#warnings-and-invariants), just in case it's useful somehow.